examples/Org.OData.Temporal.V1-history-sample.xml

```diff
@@ -0,0 +1,104 @@
+      </EntityType>
+
+      <EntityType Name="Employee_history">
+        <Key>
+          <PropertyRef Name="From" />
+        </Key>
+        <Property Name="From" Type="Edm.Date" Nullable="false" />
+        <Property Name="To" Type="Edm.Date" Nullable="false" />
+        <Property Name="Name" Type="Edm.String" Nullable="false" />
+        <Property Name="Jobtitle" Type="Edm.String" />
+        <NavigationProperty Name="Department" Type="OrgModel.Department" />
+      </EntityType>
+
+      <EntityType Name="Department">
+        <Key>
+          <PropertyRef Name="ID" />
+        </Key>
+        <Property Name="ID" Type="Edm.String" Nullable="false" />
+        <NavigationProperty Name="history" Type="Collection(OrgModel.Department_history)" ContainsTarget="true" />
+      </EntityType>
+
+      <EntityType Name="Department_history">
+        <Key>
+          <PropertyRef Name="From" />
+        </Key>
+        <Property Name="From" Type="Edm.Date" Nullable="false" />
+        <Property Name="To" Type="Edm.Date" Nullable="false" />
+        <Property Name="Name" Type="Edm.String" Nullable="false" />
+        <Property Name="Budget" Type="Edm.Decimal" />
+      </EntityType>
+
+      <EntityContainer Name="Default">
+        <EntitySet Name="Employees" EntityType="OrgModel.Employee">
+          <NavigationPropertyBinding Path="history/Department" Target="Departments" />
+        </EntitySet>
+        <EntitySet Name="Departments" EntityType="OrgModel.Department" />
+      </EntityContainer>
+
+      <Annotations Target="OrgModel.Default/Employees/history">
+        <Annotation Term="Temporal.ApplicationTimeSupport">
+          <Record>
+            <PropertyValue Property="DataHistory">
+              <Record Type="Temporal.DataHistoryTimeSlice">
+                <PropertyValue Property="TimeSliceStart" PropertyPath="history/From" />
+                <PropertyValue Property="TimeSliceEnd" PropertyPath="history/To" />
+                <PropertyValue Property="SupportedActions">
+                  <Collection>
+                    <String>Temporal.Update</String>
+                    <String>Temporal.Upsert</String>
+                    <String>Temporal.Delete</String>
+                  </Collection>
+                </PropertyValue>
+              </Record>
+            </PropertyValue>
+            <PropertyValue Property="UnitOfTime">
+              <Record Type="Temporal.UnitOfTimeDate" />
+            </PropertyValue>
+          </Record>
+        </Annotation>
+      </Annotations>
+
+      <Annotations Target="OrgModel.Default/Departments/history">
+        <Annotation Term="Temporal.ApplicationTimeSupport">
+          <Record>
+            <PropertyValue Property="DataHistory">
+              <Record Type="Temporal.DataHistoryTimeSlice">
+                <PropertyValue Property="TimeSliceStart" PropertyPath="history/From" />
```

Comment:
I believe the path evaluation rules for this `TimeSliceStart` are undefined in the spec, see [this comment](https://issues.oasis-open.org/browse/ODATA-1420?focusedCommentId=79346&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-79346).